src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -330,7 +334,22 @@ function labelFormatterBubble(config?: IChartConfig) {
     if (isNaN(value)) {
```

Comment:
try first parseFloat than you can check for isNan or better try fce isNil from lodash